### PR TITLE
EOS:21972 Added cluster_management ,sw_update & ssl_upload to the list of unsupported features

### PR DESCRIPTION
**Pull request by rohitkolapkar**
# Backend

# Problem Statement
[EOS-21972](https://jts.seagate.com/browse/EOS-21972) 

- _Overview: JIRA number/GitHub Issue added to PR: [Y/N]:_ Y
- _Describe the problem this patch intends to solve._ 

System maintenance, Software update are unsupported features.
User should not have access to these features on GUI till these features are supported in R2

## Design

- _For Bug describe the fix here._
Added below features_id to unsupported_features list
1. cluster_management
2. sw_update
3. ssl_upload

- _For feature, post the link to the solution page._
- _Any northbound interface change and has been notified to other gatekeepers [Y/N]:_

## Coding

- _Coding conventions are followed and code is consistent. Yes/No?_ Yes
- _Confirm All CODACY errors are resolved. Yes/No?_ Yes

## Testing

- _Confirm that Test Cases are added (for both the cases, fix and feature). Yes/No?_ NA
- _Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability. Yes/No?_ NA
- _Confirm Testing was performed with installed RPM. Yes/No?_ Yes
- _Confirm all the ut/regression/smoke test has been performed [Y/N]_ Yes

Unsupported features api response:
image

## Checklist

- _PR is self-reviewed? Yes/No?_ Yes
- _GitHub Issue is updated_ 
- _Jira is updated_
  -  _Check if the description is clear and explained._ 
    -  _Check Acceptance Criterion is defined._
      -  _All the tests performed should be mentioned before Resolving a JIRA._
        -  _Verification needs to be done before marked as Closed/Verified_
        - _Any interface change Yes/No?_
        - _Change notified to other gatekeepers: Yes/No?_
        - _Code reviews done and addressed: Yes/No?_
        - _Codacy issues reviewed and addressed: Yes/No?_
        - _Deployment test : Done/Not?_
        - _UT/smoke test: Done/Not?_ Done
        - _Jira number added to PR: Yes/No?_ Yes
        - _Github merge done using UI: Yes/No?_
        - _Side effects on other features - deployment/update/node-replacement_
        - _Changes to quick-start-guide/community impact_
        - _All dependent component code PR are raised or already merged Yes/No_
        - _All dependent code already merged in landing branch Yes/No_

### CODE REVIEW

**Comment by kupranay** [comment]
ok to test

**Review by udayan-y21 — APPROVED** [review]

**Review by soniyamoholkar — APPROVED** [review]

File: schema/unsupported_features.json [hunk 1]
```diff
@@ -2,27 +2,27 @@
     "setup_types": [
         {
             "name":"virtual",
-            "unsupported_features": ["lyve_pilot"]
+            "unsupported_features": ["lyve_pilot", "cluster_management", "sw_update"]
```
Comment (kupranay) [inline schema/unsupported_features.json:5]: please add ssl_upload for all types as part of unsupported list

**Review by kupranay — CHANGES_REQUESTED** [review]

File: schema/unsupported_features.json (hunk 1 above)
Comment (rohitkolapkar) [reply schema/unsupported_features.json]: @kupranay done. please review.

**Review by kupranay — APPROVED** [review]

**Comment by kupranay** [comment]
ok to test

**Pull request merged**